leasing/report/lease/invoicing_disabled_report.py

```diff
@@ -33,11 +28,31 @@ class LeaseInvoicingDisabledReport(ReportBase):
     def get_data(self, input_data):
         today = timezone.now().date()
 
-        return Lease.objects.filter(
+        qs = Lease.objects.filter(
             Q(end_date__isnull=True) | Q(end_date__gte=today),
             start_date__isnull=False,
             state__in=[LeaseState.LEASE, LeaseState.SHORT_TERM_LEASE, LeaseState.LONG_TERM_LEASE],
             is_invoicing_enabled=False
         ).select_related(
             'identifier', 'identifier__type', 'identifier__district', 'identifier__municipality',
+        ).prefetch_related(
+            'rents',
         ).order_by('start_date', 'end_date')
+
+        aggregated_data = []
+        for lease in qs:
+            print("--")
```

Comment:
I recommend you add a git pre-commit hook that runs flake8 and isort so that prints etc. don't end up in the commits.